bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -236,28 +237,28 @@ int kepler_trace(struct sched_switch_args *ctx)
     u64 on_cpu_cache_miss_delta = get_on_cpu_cache_miss(&cpu_id);
     u64 on_cpu_avg_freq = get_on_cpu_avg_freq(&cpu_id, on_cpu_cycles_delta, on_cpu_ref_cycles_delta);
     u64 on_cpu_time_delta = get_on_cpu_time(next_pid, prev_pid, cur_ts);
-
+    u64 on_cpu_time_delta_ms = on_cpu_time_delta / 1000000;
+    u64 on_cpu_time_delta_ns = (on_cpu_time_delta - on_cpu_time_delta * 1000000);
```

Comment:
@rootfs I did not understand why you need this calculation.
What it for debug? Maybe the line 241 should be removed.